_1_basics/src/main/java/code/_4_student_effort/FizzBuzz.java

```diff
@@ -0,0 +1,41 @@
+package code._4_student_effort;
+
+public class FizzBuzz {
+
+    public static void fizzBuzz(){
+        for(int i=0;i<=100;i++){
```

Comment:
daca am modifica i-ul sa mearga pana la 105, atunci 105 este divizibil si cu 3,5,7 si ar trebui sa afiseze FizzBuzzRizz

cum ai ajusta codul in cazul asta ?